canton_dep.bzl

```diff
@@ -4,5 +4,5 @@
 canton = {
     "sha": "fa0257b6b6d6673934db535216a861552ed57017481581b4f190742d13a5842e",
     "url": "https://www.canton.io/releases/canton-open-source-2.7.0-snapshot.20230719.10832.0.v93eda8bc.tar.gz",
-    "local": True,
+    "local": False,
```

Comment:
Please do not change this one. 
I know this break the local development. We will find a solution when @gary is back.

Reply:
fyi, this was change back to False in #17174